1.11/api/dcos-licensing-component-spec.yaml

```diff
@@ -0,0 +1,210 @@
+swagger: '2.0'
```

Comment:
this spec it outdated, but can be updated in a new PR.

Reply:
I don't understand what text change is needed.

Reply:
the entire yaml needs to be updated. Isabel will make a follow up PR.

Reply:
I double-checked, and there are no changes.